src/RootedTrees.jl

```diff
@@ -17,7 +17,8 @@ end
 export RootedTree, rootedtree, rootedtree!, RootedTreeIterator,
        ColoredRootedTree, BicoloredRootedTree, BicoloredRootedTreeIterator
 
-export butcher_representation, elementary_differential
+export butcher_representation, elementary_differential, elementary_differential_latexstring,
+       elementary_weight_latexstring
```

Comment:
```suggestion
export butcher_representation, elementary_differential_latexstring,
       elementary_weight_latexstring
```
We shouldn't need to `export` the deprecated name.